### PR TITLE
[API/Python] Add support for `Solver::getModel()`

### PR DESCRIPTION
**Pull request by 4tXJ7f**
This commit implements support for Solver::getModel() in the Python API,
which was missing before.

### CODE REVIEW

File: src/api/python/cvc5.pxi [hunk 1]
```diff
@@ -2117,6 +2117,35 @@ cdef class Solver:
         """
         return self.csolver.isModelCoreSymbol(v.cterm)
 
+    def getModel(self, sorts, consts):
+        """Get the model
+
+        SMT-LIB:
+
+        .. code:: smtlib
+            (get-model)
```
Comment (nafur) [inline src/api/python/cvc5.pxi:2126]:
```suggestion
        .. code:: smtlib
        
            (get-model)
```

**Review by nafur — COMMENTED** [review]
Could we add at least one test where `getModel` does not raise an exception? :)

**Comment by 4tXJ7f** [comment]
`test_get_model` contains one already (I actually just copied those tests from the C++ versions) :)

**Review by nafur — APPROVED** [review]
LGTM.

**Pull request merged**